_posts/2020-08-03-roadmap.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Roadmap
+---
+
+On June 18 2020, MDAnalysis was pleased to release the first major version, 1.0.0. As described in our [2019 roadmap](https://www.mdanalysis.org/2019/11/06/roadmap/), this is the last version that supports 2.7. We will continue backporting relevant bug fixes where feasible (e.g. the upcoming 1.0.1), but the next major release will be 2.0.0, which will support Python 3.6+.
+
+As we look forward to this next milestone, it is time to consider the next directions of MDAnalysis. The development of MDAnalysis has always been driven by the growing need for standardised, accessible analysis tools for open, reproducible, and collaborative research. While many major packages for molecular dynamics simulation provide their own set-up and analysis software, these are necessarily targeted to their own particular standards. MDAnalysis aims to provide analysis tools for simulation data in general, so historically a key objective has been to expand the number of supported package-specific data formats. As of version 1.0.0, we support 28 file formats used in major packages for both molecular dynamics and quantum chemistry. 
+
```

Comment:
Can we somehow mention the 2019 MolSSI Workshop: Molecular Dynamics Software Interoperability  https://molssi.org/2019/07/29/molssi-workshop-molecular-dynamics-software-interoperability/ e.g.,

> In 1.0.0 we also began to explore an exciting new approach: direct interoperability with other popular packages for molecular analysis by becoming **API compatible** instead of just file-format compatible, an approach also reinforced by discussions at the [2019 MolSSI Workshop: Molecular Dynamics Software Interoperability](https://molssi.org/2019/07/29/molssi-workshop-molecular-dynamics-software-interoperability/ ).

Reply:
... in addition to the report below?